store/exempleStore.ts

```diff
@@ -0,0 +1,11 @@
+export const useExempleStore = defineStore('counter', {
+  state: () => ({ count: 0, name: 'Eduardo' }),
```

Comment:
If you add names or other default values, please just fill them with something not related to a specific person. 

If you want to add a input field where users can define a name, I would recommend to fill it with "sidebase" 🤗

Reply:
i simply removed field "name"